packages/editor/src/bindings/api.js

```diff
@@ -10,6 +10,7 @@ import patternOverrides from './pattern-overrides';
 import postData from './post-data';
 import postMeta from './post-meta';
 import termData from './term-data';
+import entityUrl from './entity-url';
```

Comment:
This binding is for `entityUrl` but int he future I can imagine wanting other fields from the entity. For example `title` and binding them to other block attributes (e.g. `label`). Could we have an single binding that returns the _entire_ entity object?